src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -54,7 +54,7 @@ public MasterTestRunner(IServiceLocator services, TestPackage package)
             _runtimeService = _services.GetService<IRuntimeFrameworkService>();
             _extensionService = _services.GetService<ExtensionService>();
             _engineRunner = _services.GetService<ITestRunnerFactory>().MakeTestRunner(package);
-            LoadPackage();
+            //LoadPackage();
```

Comment:
I'm concerned that without this line we may actually cause a regression of https://github.com/nunit/nunit-console/pull/52 since `LoadPackage` is never called now